app/src/main/java/io/homeassistant/companion/android/HomeAssistantApplication.kt

```diff
@@ -274,6 +274,14 @@ open class HomeAssistantApplication : Application() {
             )
         }
 
+        // Register for changes to the configuration
+        ContextCompat.registerReceiver(
+            this,
+            sensorReceiver,
+            IntentFilter(Intent.ACTION_CONFIGURATION_CHANGED),
+            ContextCompat.RECEIVER_NOT_EXPORTED
```

Comment:
Do you want to update this with the changes in #4661?

Reply:
hah good catch! yes indeed :)